### ISSUE TITLE
Que faire pour les sites inaccessible

### ISSUE THREAD

**Issue by gcollet** [atelierPartage/wasc#5]
Lors du téléchargement, il y a des souci d'identification SSL, de Cookies, etc.
Je pensais que le module trafilatura aiderait à résoudre mais non...

Est-ce qu'on accepte qu'une partie des site est impossible à consulter automatiquement ?

**Comment by FrancoisLB** [comment]
Oui,  si un site est inaccessible, alors, il faudrait sortir un « echec de connexion » non ?

 

De : Guillaume Collet ***@***.***> 
Envoyé : lundi 10 juillet 2023 17:04
À : atelierPartage/wasc ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [atelierPartage/wasc] Que faire pour les sites inaccessible (Issue #5)

 

Lors du téléchargement, il y a des souci d'identification SSL, de Cookies, etc.
Je pensais que le module trafilatura aiderait à résoudre mais non...

Est-ce qu'on accepte qu'une partie des site est impossible à consulter automatiquement ?

**Comment by gcollet** [comment]
Oui c'est ce que je fais actuellement mais on pourrait chercher à être plus malin
